docs/UserGuide.md

```diff
@@ -106,24 +151,29 @@ It is optimized for fast typists and replicates a Command Line Interface (CLI) s
 
 </div>
 
-### 4.1. Adding your internship application:
+### 6.1. Adding your internship application:
 
-Adds your internship application to WorkBook in [sorted order](#3-things-to-note).
+Congratulations you have sent out a new internship application now you can,
+add the application to WorkBook in [sorted order](#3-things-to-note).
 
 Format: `add c/COMPANY r/ROLE s/STAGE [d/DATETIME] [e/COMPANY_EMAIL] [l/LANGUAGE TAG] [t/TAG]…​`
 
-* Date can represent:
-  * The date it happened
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** The date and time you provide could represent multiple things:<br>
+  * The date and it happened
   * The deadline of the corresponding `Stage`
+</div>
+
 
 
 Examples:
 * `add c/Meta r/Frontend Engineer s/Application Sent d/29-Oct-2022 12:00 e/hrmonkey@example.com`
 * `add c/Bytedance r/Backend Engineer s/Online Assessment d/24-Sep-2022 15:00 t/high pay l/Java l/Python`
 
-### 4.2. Editing your internship application:
+### 6.2. Editing your internship application:
 
-Edits your existing internship application in the WorkBook.
+In the scenario that your internship application has progressed to a different stage and requires updating this command,
```

Comment:
```suggestion
Good job! You've passed the Resume Screening and have been invited to complete an Online Assessment for your internship application. Now's a good time to use this command to update the application so that WorkBook can continue to give you relevant tips and reminders!
```